name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/SantanderConsumerBankPDFExtractor.java

```diff
@@ -256,6 +256,31 @@ private void addAccountStatementTransaction()
                             t.setNote(TextUtil.replaceMultipleBlanks(v.get("note")).trim());
                         })
 
+                        .wrap(TransactionItem::new));
+
+        // 20.05.2023 20.05.2023 -1,00 33,01 Auszahlung auf ...
+        Block removalBlock = new Block(
+                        "^(?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) ([\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) (?<amount>[\\.,\\d-]+) ([\\.,\\d-]+) (Auszahlung) (?<note>.*)");
+        type.addBlock(removalBlock);
+        removalBlock.set(new Transaction<AccountTransaction>()
+
+                        .subject(() -> {
+                            AccountTransaction entry = new AccountTransaction();
+                            entry.setType(AccountTransaction.Type.REMOVAL);
+                            return entry;
+                        })
+
+                        .section("date", "amount", "note")
+                        .match("^(?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) ([\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) (?<amount>[-\\.,\\d-]+) ([\\.,\\d-]+) (Auszahlung) (?<note>.*)")
+                        .assign((t, v) -> {
+                            Map<String, String> context = type.getCurrentContext();
+
+                            t.setDateTime(asDate(v.get("date")));
+                            t.setAmount(asAmount(v.get("amount")));
+                            t.setCurrencyCode(asCurrencyCode(context.get("currency")));
+                            t.setNote(TextUtil.replaceMultipleBlanks(v.get("note")).trim());
```

Comment:
You are already using static import of functions.
- Use static imports... see [Contributing rules --> Configure Eclipse](https://github.com/buchen/portfolio/blob/master/CONTRIBUTING.md#configure-eclipse)
- You need to add on top --> If your Eclipse is configured correctly, the imports will be added automatically.
   `import static name.abuchen.portfolio.util.TextUtil.trim;` <-- This already exists in Line 4.
   `import static name.abuchen.portfolio.util.TextUtil.replaceMultipleBlanks;`  <-- This must be added.

```suggestion
                            t.setNote(trim(replaceMultipleBlanks(v.get("note"))));
```